openedx/core/djangoapps/team_api/models.py

```diff
@@ -0,0 +1,88 @@
+"""
+Django models related to teams functionality.
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from .utils import slugify
+
+from xmodule_django.models import CourseKeyField
+
+
+class CourseTeam(models.Model):
+    """
+    This model represents team related info.
+    """
+
+    class Meta:
+        unique_together = (('team_id', 'course_id'),)
+
+    team_id = models.CharField(max_length=100, db_index=True)
```

Comment:
Why is this "team_id" vs. just "id"? i think it would make more sense for it to just be "id", and that is what is in the spec.